Soundex.Tests.cpp

```diff
@@ -7,3 +7,75 @@ TEST(SoudexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
   generateSoundex("AX", soundex);
  // ASSERT_EQ(soundex,"A200");
 }
+TEST(SoudexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
+ 
```

Comment:
Your Build Error because of using "ReplacesConsonantsWithAppropriateDigits" as name for every test case
TEST() macro accept two parameter 
1. TestSuiteName
2. TestCaseName

One Test Suite Can have Multiple TestCases
Choose appropriate TestCase Name for each test cases